### ISSUE TITLE
vscode could not veirfy lombok word in latest version

### ISSUE THREAD

**Issue by bobogavin** [microsoft/vscode-lombok#80]
vscode could not veirfy lombok word in latest version

**Comment by bobogavin** [comment]
vscode stable version and insider version both have the problem

**Comment by bobogavin** [comment]
vscode version 1.68.1

**Comment by bobogavin** [comment]
Version: 1.68.1 (user setup)
Commit: 30d9c6cd9483b2cc586687151bcbcd635f373630
Date: 2022-06-14T12:48:58.283Z
Electron: 17.4.7
Chromium: 98.0.4758.141
Node.js: 16.13.0
V8: 9.8.177.13-electron.0
OS: Windows_NT x64 10.0.19044

**Comment by bobogavin** [comment]
![Uploading image.png…]()

**Comment by bobogavin** [comment]
![image](https://user-images.githubusercontent.com/17951661/177000585-f09714f0-8d3b-4615-9200-41a9bf4231fb.png)

**Comment by pinkli** [comment]
yes I have the same issue.

**Comment by zmwxiaoming** [comment]
yes I have the same issue.

**Comment by zmwxiaoming** [comment]
版本: 1.68.1
提交: 30d9c6cd9483b2cc586687151bcbcd635f373630
日期: 2022-06-14T12:52:13.188Z (2 周前)
Electron: 17.4.7
Chromium: 98.0.4758.141
Node.js: 16.13.0
V8: 9.8.177.13-electron.0
OS: Darwin x64 20.3.0

**Comment by leegohi** [comment]
I solve the problem.
first ,find the project lombok version
`mvn help:effective-pom  | grep lombok -A5 -B5`
I find it is 1.18.2
but when I look vscode lombok version which version is 1.18.12
so,I change my project lombok version  in pom.xml to 1.18.12. it's solved.
`<dependency>
        <groupId>org.projectlombok</groupId>
        <artifactId>lombok</artifactId>
        <version>1.18.12</version>`

**Comment by Arisaematis** [comment]
yes I have the same issue.

**Comment by zmwxiaoming** [comment]
> I solved the problem that costs one day!!! First ,show the project lombok version: `mvn help:effective-pom | grep lombok -A5 -B5` I find it is 1.18.2 but when I look vscode lombok version which version is 1.18.12 so,I change my project lombok version in pom.xml to 1.18.12. it's solved. `<dependency> <groupId>org.projectlombok</groupId><artifactId>lombok</artifactId> <version>1.18.12</version> </dependency>`

very good !

**Comment by pinkli** [comment]
> I solved the problem that costs one day!!! First ,show the project lombok version: `mvn help:effective-pom | grep lombok -A5 -B5` I find it is 1.18.2 but when I look vscode lombok version which version is 1.18.12 so,I change my project lombok version in pom.xml to 1.18.12. it's solved. `<dependency> <groupId>org.projectlombok</groupId><artifactId>lombok</artifactId> <version>1.18.12</version> </dependency>`

Yes, after updating the lombok version to 1.18.12 in pom.xml file, vscode works.

**Comment by zmwxiaoming** [comment]
you also can see 
https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-lombok
Requirements
VS Code (version 1.21.0 or later)
Lombok added as a dependency in your Java Project (Make sure you're using the latest version to avoid issues!) [Add with Maven](https://projectlombok.org/setup/maven) or [Add with Gradle](https://projectlombok.org/setup/gradle)

**Comment by testforstephen** [comment]
[Language Support for Java(TM) by RedHat](https://marketplace.visualstudio.com/items?itemName=redhat.java) extension 1.8.0 will automatically use the lombok.jar from your project classpath for lombok support in VS Code. Based on testing, it seems the lombok version compatible with RedHat Java extension is **1.18.4** and higher.

If the lombok dependency in your project pom.xml is lower than 1.18.4, the lombok annotations may not work in VS Code. You need to update lombok version to a newer one.

**Comment by VincentLee-EN** [comment]
> ![image](https://user-images.githubusercontent.com/17951661/177000585-f09714f0-8d3b-4615-9200-41a9bf4231fb.png)

+1

**Comment by VincentLee-EN** [comment]
> vscode lombok version

how to check vscode lombok version?
